engine_wrapper.py

```diff
@@ -118,13 +119,11 @@ def print_stats(self, board):
 
     def get_stats(self, board, for_chat=False):
         info = self.last_move_info.copy()
-        if "pv" not in info:
-            info["pv"] = []
         if for_chat:
             stats = ["depth", "nps", "nodes", "score", "ponderpv"]
```

Comment:
The `stats` variable is the same in both branches of `if for_chat:`. Put this line above the `if` statement and you can delete the `else` branch.

Reply:
Done. It is not ready to be merged. I will change a few more things (e.g. board in get_stats is not used).